products/spring-boot.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Spring Boot
+category: framework
+permalink: /spring-boot
+alternate_urls:
+-   /springboot
+-   /sb
+releasePolicyLink: https://github.com/spring-projects/spring-boot/wiki/Supported-Versions
+changelogTemplate: "https://github.com/spring-projects/spring-boot/releases/tag/v__LATEST__"
+auto:
+-   git: https://github.com/spring-projects/spring-boot.git
+# See https://rubular.com/r/XQUdQN2MHdmmCD for reference
+    regex: '^v?(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.(?<patch>0|[1-9]\d*)(\.RELEASE)?$'
+releases:
+-   releaseCycle: "3.0"
+    eol: 2023-11-24
+    support: 2025-02-24
+    latest: "3.0.0-RC1"
+    latestReleaseDate: 2022-11-24
+    releaseDate: 2022-11-24
+-   releaseCycle: "2.7"
+    eol: 2023-11-18
+    support: 2025-02-18
+    latest: "2.7.5"
+    latestReleaseDate: 2022-10-21 
+    releaseDate: 2022-05-19
+-   releaseCycle: "2.6"
+    eol: 2022-11-24
+    support: 2024-02-24
+    latest: "2.6.13 "
+    latestReleaseDate: 2022-10-20 
+    releaseDate: 2021-11-17
+-   releaseCycle: "2.5"
+    eol: 2022-05-19
+    support: 2023-08-24
+    latest: "2.5.14 "
+    latestReleaseDate: 2022-05-19 
+    releaseDate: 2021-05-20
+-   releaseCycle: "2.4"
+    eol: 2021-11-18
+    support: 2023-02-23
+    latest: "2.4.13 "
+    latestReleaseDate: 2021-11-18 
+    releaseDate: 2020-11-12
+-   releaseCycle: "2.3"
+    eol: 2021-05-20
+    support: 2022-08-20
+    latest: "v2.3.12"
+    latestReleaseDate: 2021-06-10 
+    releaseDate: 2020-05-15
+-   releaseCycle: "2.2"
+    eol: 2020-10-16
+    support: 2022-01-16
+    latest: "v2.2.13"
```

Comment:
some of them have `v` in their latest tag some doesnt

Reply:
i suggest we remove all `v` and add it through out changelogtemplate ( which you allready done )

Reply:
Sure :+1: